extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/handler/TradfriLightHandler.java

```diff
@@ -147,6 +148,17 @@ public void onUpdate(JsonElement data) {
                     , state.getLightId(), dimmer, colorTemp, devicefirmware, modelId, vendor);
         }
     }
+    
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        super.bridgeStatusChanged(bridgeStatusInfo);
+
+        if (bridgeStatusInfo.getStatus() == ThingStatus.ONLINE) {
+            scheduler.schedule(() -> {
+                coapClient.startObserve(this);
+            }, 3, TimeUnit.SECONDS);
```

Comment:
Why should we wait 3 seconds here?

Reply:
I re-used the logic from `initialize()` which to me makes sense.

Reply:
Imho in initialize this is done because the coap client is instantiated right before, so it gives it some time before doing the first request.
At this code here, we do not do any changes to the coap client instance, so I do not see any reason to postpone the observe request.